libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -810,8 +828,15 @@ bool ContractCompiler::visit(EmitStatement const& _emit)
 
 bool ContractCompiler::visit(VariableDeclarationStatement const& _variableDeclarationStatement)
 {
-	StackHeightChecker checker(m_context);
 	CompilerContext::LocationSetter locationSetter(m_context, _variableDeclarationStatement);
+
+	// Local variable slots are reserved when their declaration is seen,
```

Comment:
`visited` instead of `seen`? (seen could refer to "being in the same scope" or something like that)